cypress/integration/delete-collection.ts

```diff
@@ -0,0 +1,24 @@
+describe("Let's delete from collection", () => {
+  it('Adds cards from search page', () => {
+    cy.visit('/')
+    cy.contains('Search').click()
```

Comment:
```suggestion
    cy.findByRole('link', { name: 'Search' }).click()
```

Make sure your always opting to use cypress-testing-library when querying and not the default Cypress stuff.

You also want to try to query by aria roles when you can (that way we're testing accessibility at the same time!)